lib/index.js

```diff
@@ -18,6 +18,25 @@ module.exports.SyntaxError = Parser.SyntaxError;
 var astutil = require('./astutil');
 var contains_func = module.exports.contains_func = astutil.contains_func;
 
+function handleTexError(e, options) {
```

Comment:
nice. You should also do a pull requst for this change in the (upstream repo)[https://github.com/wikimedia/texvcjs].